autosklearn/smbo.py

```diff
@@ -536,17 +539,39 @@ def get_metalearning_suggestions(self):
                                   else 'dense'))
                 self.metadata_directory = metadata_directory
 
+            # If metadata directory is specified by user,
+            # then verify that it exists.
+            else:
+                if not os.path.exists(self.metadata_directory):
+                    raise ValueError('The specified metadata directory \'%s\' '
+                                     'does not exist!' % self.metadata_directory)
+
+                else:
+                    # There is no multilabel data in OpenML
+                    if self.task == MULTILABEL_CLASSIFICATION:
+                        meta_task = BINARY_CLASSIFICATION
+                    else:
+                        meta_task = self.task
+
+                    metadata_directory = os.path.join(
+                        self.metadata_directory,
+                        '%s_%s_%s' % (self.metric, TASK_TYPES_TO_STRING[meta_task],
+                                      'sparse' if self.datamanager.info['is_sparse']
+                                      else 'dense'))
+                    # Check that the metadata directory has the correct
+                    # subdirectory needed for this dataset.
+                    if metadata_directory not in os.listdir(self.metadata_directory):
```

Comment:
Could you please also add the subdirectory you're checking for to the error message?

Reply:
Yep, I will modify the error message such that it explicitly prints the required subdirectory if it does not exist. After checking that travis runs successfully, I will merge it.